sagemaker_neo_compilation_jobs/gluoncv_yolo_darknet/gluoncv_yolo_darknet_neo.ipynb

```diff
@@ -0,0 +1,609 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "compiled_model = yolo_estimator.compile_model(target_instance_family='ml_p3', \n",
+    "                                             input_shape={'data':[1, 3, data_shape, data_shape]},\n",
+    "                                             output_path=s3_output_location,\n",
+    "                                             framework='mxnet', \n",
+    "                                             framework_version='1.4.1'\n",
+    "                                            )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sagemaker.predictor import RealTimePredictor\n",
+    "compiled_model.predictor_cls = RealTimePredictor"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "object_detector = compiled_model.deploy(initial_instance_count = 1,\n",
+    "                                        instance_type = 'ml.p3.2xlarge')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Inference\n",
+    "Now that the trained model is deployed at an endpoint that is up-and-running, we can use this endpoint for inference. To do this, we use an image from [PEXELS](https://www.pexels.com/) which the algorithm has so-far not seen. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "file_name = 'test.jpg'\n",
+    "\n",
+    "with open(file_name, 'rb') as f:\n",
+    "    payload = f.read()\n",
+    "    payload = bytearray(payload) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let us use our endpoint to try to detect objects within this image. Since the image is `jpeg`, we use the appropriate `content_type` to run the prediction job. The endpoint returns a JSON file that we can simply load and peek into."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "object_detector.content_type = 'image/jpeg'\n",
+    "response = object_detector.predict(payload)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "detections = json.loads(response)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The format of the output can be represented as `[class_index, confidence_score, xmin, ymin, xmax, ymax]`. Typically, we don't consider low-confidence predictions."
+   ]
+  },
+  {
+   "cell_type": "markdown",
```

Comment:
set "cell_type" to "code" in order to define the neo_visualize_detection() function

Re: the next code block needs this function

Reply:
Thanks, I missed that. Done.